### PR TITLE
GH-974: update pretrained models

### LINKED ISSUE #974: Replace pre-trained models with deprecated embeddings
There's a list of pre-trained models in Flair that still use the deprecated `CharLMEmbeddings` and may suffer from the dropout bug. These models are (_x_ marks dropout bug) and need to be retrained: 

`SequenceTagger`
- [x] 'ner-fast' (x)
- [x] 'ner-ontonotes'
- [x] 'ner-ontonotes-fast'
- [x] 'pos-fast' (x)
- [x] 'frame' (x)
- [x] 'frame-fast' (x)
- [x] 'chunk-fast' (x)
- [x] 'de-pos' (x)
- [x] 'de-ner'
- [x] 'fr-ner'

`TextClassifier`
- [x]  'en-sentiment'
- [x] 'de-offensive-language'

### PR DESCRIPTION
**Pull request by alanakbik**
This PR updates all pretrained models that used embeddings that are now deprecated. 

Closes #974

### CODE REVIEW

**Comment by yosipk** [comment]
👍

**Comment by alanakbik** [comment]
:+1:
